nbdime/tests/test_git_diffdriver.py

```diff
@@ -29,7 +31,8 @@
 \ No newline at end of file
 +foo(1, 2)
 \ No newline at end of file
```

Comment:
We should probably remove `\ No newline at end of file` from output + expected output here?